mava/configs/env/gigastep.yaml

```diff
@@ -0,0 +1,13 @@
+# ---Environment Configs---
+defaults:
+  - _self_
+  - scenario: hide_and_seek_5_vs_5_pobs # [hide_and_seek_5_vs_5_fobs, hide_and_seek_5_vs_5_pobs, waypoint_5_vs_5_fobs, waypoint_5_vs_5_pobs]
```

Comment:
Please mention that fobs and pobs stand for full and partial observation respectively.